libpod/logs/log.go

```diff
@@ -120,11 +122,16 @@ func getTailLog(path string, tail int) ([]*LogLine, error) {
 func (l *LogLine) String(options *LogOptions) string {
 	var out string
 	if options.Multi {
-		cid := l.CID
-		if len(cid) > 12 {
-			cid = cid[:12]
+		if options.Names {
```

Comment:
@rhatdan I am confused about this option. Could you please provide some comments here?